Proxies/Array.cs

```diff
@@ -21,25 +21,43 @@ public abstract class ArrayProxy {
         [JSAlwaysAccessAsProperty]
         [JSNeverReplace]
         [JSIsPure]
-        abstract public long LongLength { get; } 
+        abstract public long LongLength { get; }
 
 
+        [JSReplacement("$$jsilcore.System.Array.prototype.GetLowerBound.call($this, $dimenstion)")]
+        public int GetLowerBound(int dimenstion)
+        {
+            throw new InvalidOperationException();
+        }
+
+        [JSReplacement("$$jsilcore.System.Array.prototype.GetUpperBound.call($this, $dimenstion)")]
+        public int GetUpperBound(int dimenstion)
+        {
+            throw new InvalidOperationException();
+        }
+
+        [JSReplacement("$$jsilcore.System.Array.prototype.GetLength.call($this, $dimenstion)")]
+        public int GetLength(int dimenstion)
+        {
+            throw new InvalidOperationException();
+        }
+
         [JSReplacement("JSIL.Array.New($elementType, $size)")]
         public static System.Array CreateInstance (Type elementType, Int32 size) {
             throw new InvalidOperationException();
         }
 
-        [JSReplacement("JSIL.MultidimensionalArray.New.apply(null, [$elementType].concat($sizes))")]
+        [JSReplacement("JSIL.MultidimensionalArray.CreateInstance(elementType, $sizes)")]
```

Comment:
I'm confused how one of these is MultidimensionalArray.CreateInstance and the following ones are .New.